docs/platforms/python/configuration/environments.mdx

```diff
@@ -6,10 +6,10 @@ description: "Learn how to configure your SDK to tell Sentry about your environm
 
 Environments tell you where an error occurred, whether that's in your production system, your staging server, or elsewhere.
 
-Sentry automatically creates an environment when it receives an event with the `environment` parameter set.
-
 Environments are case-sensitive. The environment name can't contain newlines, spaces or forward slashes, can't be the string "None", or exceed 64 characters. You can't delete environments, but you can [hide](/product/sentry-basics/environments/#hidden-environments) them.
 
 <PlatformContent includePath="set-environment" />
 
+The `environment` can also be set by the environment variable, `SENTRY_ENVIRONMENT`. If no `environment` is set, it will default to `production`.
+
```

Comment:
I think what would be really great is to also mention, what takes precedence (env var or config etc).

Reply:
Addressed here: https://github.com/getsentry/sentry-docs/pull/9473/files